web/src/routes/api/leagues/[leagueId]/standings/+server.ts

```diff
@@ -0,0 +1,67 @@
+import { supabase } from '$lib/db/supabase/supabase-client';
+import { error, json } from '@sveltejs/kit';
+
+// TODO: type the return value
+async function getStandings(leagueId: bigint, startWeek: number = 1, endWeek: number = 17) {
+  const { data, error } = await supabase
+    .from('schedule')
+    .select(
+      `
+      team_id, 
```

Comment:
It doesn't matter, but you don't need to request the `team_id` column from the table for this type of query. More of a best practice thing, you shouldn't request any column you don't need to use (it has no bearing on computation time or anything, just extra info pulled into memory that isn't necessary when you make a query)